### PR TITLE
g2-2024-v7-#15808

### PR DESCRIPTION
**Pull request by a22eddpe**

### CODE REVIEW

**Review by a22albjo — CHANGES_REQUESTED** [review]
Im getting error alerts in navheader where ever i go on the page:
Skärmavbild 2024-05-15 kl  16 16 01
Skärmavbild 2024-05-15 kl  16 18 00

Other than that i can't find any other problems.

**Review by a22albjo — CHANGES_REQUESTED** [review]
Many of the services are outdated and therefor hard to test, many of the services are/has been included in DuggaSys files for example sectioned.js.
Path: LenaSys/DuggaSys/sectioned.js

Many of the files that has the changed includes are in the folder Shared for example:
Path: LenaSys/Shared/announcementBox.php

announcementBox had this code
`include_once ("../Shared/database.php");`

When announcementBox is included into the 
LenaSys/DuggaSys/sectioned.js
It went back one step to LenaSys then forward to Shared and then found database.php.
Now using:
include_once "./database.php"; when announcementBox.php is included into sectioned.js it will look for the database.php in the DuggaSys folder and wont find it.

To solve this you can for example use something like:
include_once(__DIR__ . "/database.php");
Where dir always is the folder where the file is included. This will mean that it will always look in the right place where ever it is included.

**Comment by a22eddpe** [comment]
i need to look at the orginal files, and where the includes looks like this "../Shared etc, i will use this a DIR. This makes them stand out from the other includes and hopefully reduces the likelihood that those with DIR includes are confused with regular includes.

File: Shared/sessions.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
-require_once('database.php');
-require_once('constants.php');
+require_once(dirname(__FILE__) . '/../Shared/database.php');
```
[the remark this answers is not in the document]
Comment (a22eddpe) [reply Shared/sessions.php:2]: don't know. This was what the original path therefore i left it as it was

File: Shared/sessions.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+require_once(dirname(__FILE__) . '/constants.php');
```
[the remark this answers is not in the document]
Comment (a22eddpe) [reply Shared/sessions.php:3]: same with this

**Pull request merged**